kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/service/impl/B2BPurchaseOrderSciquestServiceImpl.java

```diff
@@ -241,6 +267,12 @@ public String getCxml(PurchaseOrderDocument purchaseOrder, String requisitionIni
 
         return cxml.toString();
     }
+    
+    protected KualiDecimal calcTaxRate(KualiDecimal preTaxAmount, KualiDecimal taxAmount) {
+        if (taxAmount == null || preTaxAmount == null) { return KualiDecimal.ZERO; }
+        if (KualiDecimal.ZERO.equals(preTaxAmount)) { return KualiDecimal.ZERO; }

```

Comment:
Any reason why these three conditions are separated into two conditionals? (On an aside, the one-liner 'if' hurts readability since the convention is not consistent in this file... if you find that compelling, I'd encourage you to change it, though not found in any of our style guides I could find).